spaceship/templates/email_mission_end.html

```diff
@@ -0,0 +1,26 @@
+
+<h1>LOGO</h1>
+
+<h3>Hi Team {{ mission.team.name }}!</h3>
+
+<p>
+Our mission to {{ mission.short_description }} is now complete.
+Time to check in to see how you did!
+</p>
+
+<p>
+Please fill out the post-mission survey here:
```

Comment:
We can link to some route like /crew/id/survey?mission=mission_id and redirect to (or embed) a google form there?

Seems like it might be possible to pre-fill the google form via url params:
https://trevorfox.com/2015/06/dynamically-pre-fill-google-forms-with-mailchimp-merge-tags/